datalad/utils.py

```diff
@@ -1546,56 +1546,66 @@ def import_modules(mods, pkg, msg="Failed to import {module}", log=lgr.debug):
     """
     from importlib import import_module
     _globals = globals()
-    for mod in mods:
+    mods_loaded = []
+    for modname in modnames:
         try:
-            _globals[mod] = import_module(
-                '.{}'.format(mod),
+            _globals[modname] = mod = import_module(
+                '.{}'.format(modname),
                 pkg)
+            mods_loaded.append(mod)
```

Comment:
ha ha -- apparently `import_modules` is not used anywhere and the test only tests for "bogus" case thus this line is never reached.  I will test it a bit better while preparing the test just so we have it available happen we need it